clients/src/main/java/org/apache/kafka/common/protocol/MessageUtil.java

```diff
@@ -204,13 +204,17 @@ public static boolean compareRawTaggedFields(List<RawTaggedField> first,
         }
     }
 
-    public static ByteBuffer toByteBuffer(final Message message, final short version) {
+    public static ByteBufferAccessor toReadable(final Message message, final short version) {
         ObjectSerializationCache cache = new ObjectSerializationCache();
         int messageSize = message.size(cache, version);
         ByteBufferAccessor bytes = new ByteBufferAccessor(ByteBuffer.allocate(messageSize));
         message.write(bytes, cache, version);
         bytes.flip();
-        return bytes.buffer();
+        return bytes;
+    }
+
+    public static ByteBuffer toByteBuffer(final Message message, final short version) {
```

Comment:
Is this used anywhere?

Reply:
Yes. In a bunch of tests and few other classes.